src/test/java/net/openhft/chronicle/threads/EventGroupTest.java

```diff
@@ -588,7 +588,7 @@ public boolean action() throws InvalidEventHandlerException {
 
         @Override
         public void loopStarted() {
-            assertTrue(loopStartedNS.compareAndSet(0, System.nanoTime()), "loopStarted should only ever be called once");
+            assertTrue(loopStartedNS.compareAndSet(0, System.nanoTime()), "loopStarted should only ever be called once " + this);
```

Comment:
unrelated as well but this test failed on the first push. Haven't been able to reproduce it but added some more info.